ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/GetAdminUserWithGivenUserIdAuthIT.scala

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.sandbox.auth
+
+import com.daml.ledger.api.v1.admin.user_management_service._
+import io.grpc.{Status, StatusRuntimeException}
+
+import java.util.UUID
+import scala.concurrent.Future
+
+class GetAdminUserWithGivenUserIdAuthIT extends AdminServiceCallAuthTests {
+  override def serviceCallName: String = "UserManagementService#GetUser(given-user-id)"
+
+  // admin and idp admin users are allowed to specify a user-id other than their own for which to retrieve a user
+  override def serviceCall(context: ServiceCallContext): Future[Any] = {
+    val testId = UUID.randomUUID().toString
+
+    def getUser(userId: String): Future[User] =
+      stub(UserManagementServiceGrpc.stub(channel), context.token)
+        .getUser(GetUserRequest(userId))
+        .map(_.user.get)
+
+    for {
+      // create a normal users
+      (alice, _) <- createUserByAdmin(testId + "-alice")
+
+      // test that only admins can retrieve his own user and the newly created alice user
+      _ <- getUser("participant_admin")
```

Comment:
This doesn't test that _only_ the admin can do that, does it?

Reply:
removed this class